src/components/app/App.tsx

```diff
@@ -1,30 +1,29 @@
-import bugsnag, { Bugsnag } from "@bugsnag/js";
+import * as React from "react";
+import bugsnag from "@bugsnag/js";
 import bugsnagReact from "@bugsnag/plugin-react";
-import {
-  AppBar,
-  createMuiTheme,
-  IconButton,
-  Link,
-  Menu,
-  MenuItem,
-  Toolbar,
-  Typography
-} from "@material-ui/core";
-import * as MenuIcon from "@material-ui/icons/Menu";
+
+// Bugsnag docs recommend instantiating the BugsnagClient as soon as possible and before the rest of the module's imports
+const bugsnagClient = bugsnag({
+  apiKey: "2d77ffd5445ab337cd3c996a2400438a",
+  notifyReleaseStages: ["production"]
```

Comment:
Should this vary depend on whether we're in dev mode? Though I don't think we currently have different release stages defined, so this may be a moot point

Reply:
what `notifyReleaseStages` does is only log things to bugsnag if `process.env.NODE_ENV` is equal to `production`. When CRA builds I believe it automatically sets NODE_ENV to `production`...but i could be wrong. The code above will for sure work as expected as soon as I use `env-cmd` to set environment. In the meantime i wanted to make sure that no reporting to bugsnag was being done in dev, test or staging environment.